javascript/javascript1/week2/main.js

```diff
@@ -0,0 +1,149 @@
+//Flight booking fullname function
+function getFullName(
+  firstname = "Not provided",
+  surname = "Not provided",
+  useFormalName,
+  male
+) {
+  if (useFormalName) {
+    male
+      ? console.log(`Lord ${firstname} ${surname}`)
+      : console.log(`Mis ${firstname} ${surname}`);
+  } else {
+    console.log(`${firstname} ${surname}`);
+  }
+}
+const fullname1 = getFullName("Jan", "Jayson", true, true);
+const fullname2 = getFullName("Peter", "Petersen", false);
+const fullname3 = getFullName("Alex", "Ferguson"); // if useFormalName is not given, consider it as false.
+const fullname4 = getFullName("Maria", "Andersen", true, false);
+const fullname5 = getFullName("", "", true, false); // if there is empty string, no firstname and surenam is typed.
+
+
+//vent application
+function getEventWeekday(dayNumber) {
+  const weekDays = [
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+    "Sunday",
+  ];
+
+  const dayOfWeek = new Date().getDay();
+  const eventIndex = (dayOfWeek + dayNumber) % 7;
+
+  if (eventIndex === 0) {
+    console.log(`The day of the event is: Sunday`);
+  } else {
+    console.log(`The day of the event is: ${weekDays[eventIndex - 1]}`);
+  }
```

Comment:
You may want to read how .getDay works in order to simplify your function and change how weekDays are created